include/AnalogSensor.h

```diff
@@ -7,10 +7,13 @@
 class AnalogSensor : public Sensor
 {
     uint8_t pin = 0;
+    unsigned long long runner = 0; // Lots of buffer
```

Comment:
IDK what the counter and runner are for, I can't find their usage anywhere. Is it used in DCR or ECU? What's the idea behind them?

Reply:
Left over code from another PR. Was gonna do both together but gonna do separate. Will remove